frontend/src/modules/SimulationTimeSeriesSensitivity/view.tsx

```diff
@@ -1,24 +1,29 @@
 import React from "react";
 
-import { VectorRealizationData_api } from "@api";
-import { StatisticFunction_api } from "@api";
-import { BroadcastChannelMeta } from "@framework/Broadcaster";
+import { StatisticFunction_api, VectorRealizationData_api, VectorStatisticSensitivityData_api } from "@api";
+import { BroadcastChannelMeta, BroadcastChannelData } from "@framework/Broadcaster";
+import { Ensemble } from "@framework/Ensemble";
 import { ModuleFCProps } from "@framework/Module";
+import { useSubscribedValue } from "@framework/WorkbenchServices";
+import { timestampUtcMsToCompactIsoString } from "@framework/utils/timestampUtils";
 import { useElementSize } from "@lib/hooks/useElementSize";
 
 import { indexOf } from "lodash";
 
 import { BroadcastChannelNames } from "./channelDefs";
 import { useStatisticalVectorSensitivityDataQuery, useVectorDataQuery } from "./queryHooks";
-import { TimeSeriesChart } from "./simulationTimeSeriesChart/chart";
-import {
-    TimeSeriesPlotlyTrace,
-    createRealizationLineTraces,
-    sensitivityStatisticsTrace,
-} from "./simulationTimeSeriesChart/traces";
+import { HoverInfo, TimeSeriesChart } from "./simulationTimeSeriesChart/chart";
+import { TimeSeriesPlotlyTrace } from "./simulationTimeSeriesChart/traces";
+import { createRealizationLineTraces, createSensitivityStatisticsTrace } from "./simulationTimeSeriesChart/traces";
```

Comment:
Splitting functions/types on purpose into two different imports? If yes, why?

Reply:
+1 for readability in my opinion. I'll leave it but we can pick it up later.